pytest.ini

```diff
@@ -19,8 +19,8 @@
 addopts =
     -rasl
     --verbosity=2
-;    This will treat all tests as flaky
-;    --force-flaky
+    ; Disable `flaky` plugin for pytest. This plugin conflicts with `rerunfailures` because provide same marker.
+    -p no:flaky
```

Comment:
Should we explicitly depend on `pytest-rerunfailures`?

Reply:
Yeah, out dev dependencies already depend on `pytest-rerunfailures`

https://github.com/apache/airflow/blob/4e545c6e54712eedb6ca9cbb8333393ae3f6cba2/setup.py#L402-L404

I do not know how we use this plugin before - it just sits in our dev-requirements, the same with `flaky` it is only used for couple tests, which can replaced by `pytest-rerunfailures`

There is maybe another good question: should we specify explicitly specify all plugins which we actually use? But this only help to fail fast if plugin not installed

Reply:
If that works no issues, was just wondering if it's possible to add
```
required_plugins = 
    pytest-rerunfailures
```

Reply:
```console
========================================================= test session starts =========================================================
platform linux -- Python 3.7.16, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /opt/airflow, configfile: pytest.ini
plugins: cov-4.0.0, asyncio-0.20.3, rerunfailures-9.1.1, instafail-0.4.2, anyio-3.6.2, timeouts-1.2.1, xdist-3.1.0, requests-mock-1.10.0, capture-warnings-0.0.4, httpx-0.21.2, time-machine-2.8.2
```

THis work without **required_plugins** but no problem add this

Reply:
Ohhh... we use same config for breeze tests

```console
Run python -m pytest ./dev/breeze/ -n auto --color=yes
ERROR: Missing required plugins: pytest-rerunfailures

Error: Process completed with exit code 4.
```